data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/CustomPluginMapper.java

```diff
@@ -0,0 +1,10 @@
+package org.opensearch.dataprepper.logstash.mapping;
+
+import com.amazon.dataprepper.model.configuration.PluginModel;
+import org.opensearch.dataprepper.logstash.model.LogstashPlugin;
+
+import java.util.List;
+
+public interface CustomPluginMapper {
```

Comment:
Please add Javadocs (including `@since`) for this class and the `mapPlugin` method since they are public.